core/src/test/java/io/cucumber/core/plugin/JUnitFormatterTest.java

```diff
@@ -76,7 +78,7 @@ public void should_format_passed_scenario() throws Throwable {
         stepsToResult.put("first step", result("passed"));
         stepsToResult.put("second step", result("passed"));
         stepsToResult.put("third step", result("passed"));
-        stepDurationMillis = 1L;
+        stepDuration = Duration.ofNanos(1L);
```

Comment:
Why change from milis to nanos here?

Reply:
Nothing in particular but since I have been using nanos everywhere with the new `Duration` object for the sake of precision I updated this to that as well, along with the formatter too.

You want me to revert it to millis?